README.md

```diff
@@ -339,6 +373,31 @@ https://testing.imgix.net/image.jpg?w=100&dpr=4&q=23 4x,
 https://testing.imgix.net/image.jpg?w=100&dpr=5&q=20 5x
 ```
 
+Quality parameters is overridable for each `dpr` by passing `targetDPRRatiosQualities` parameters.
+
+```js
+const client = new ImgixClient({
+  domain: 'testing.imgix.net',
+  includeLibraryParam: false,
+});
+
+const srcset = client.buildSrcSet(
+  'image.jpg',
+  { w: 100 },
+  { targetDPRRatiosQualities: { 1: 45, 2: 30, 3: 20, 4: 15, 5: 10 } },
+);
```

Comment:
Nitpick, Let's add a `log` here to keep it consistent with the previous example.

```suggestion
);

console.log(srcset);
```